crm_lead_sale_link/README.rst

```diff
@@ -0,0 +1,56 @@
+CRM Lead Sale Link
+==================
+
+This module extends the functionality of crm to deliver two contexted buttons into the opportunity form view :
+    the first opens all quotes linked to this opportunity
+    the second opens all sales order linked to this opportunity
+
+Installation
+============
+
+To install this module, you just need to select the module and insure yourself dependancies are available.
+
+Configuration
+=============
+
+No configuration to use this module.
+
+Usage
+=====
+
+To use this module, you need to :
+- create a lead
+- transform it into an opportunity
+- create a quote from this opportunity by using the button
+- you should get back and keep link beetween the models
+
+Known issues / Roadmap
+======================
+
+Credits
+=======
+
+Module developed and tested with Odoo version 8.0
+For questions, please contact our support services <support@savoirfairelinux.com>
+
+Contributors
+------------
+
+David DUFRESNE <david.dufresne@savoirfairelinux.com>
+Jordi RIERA <jordi.riera@savoirfairelinux.com>
+Bruno JOLIVEAU <bruno.joliveau@savoirfairelinux.com>
+
++Icon
++----
++* http://en.wikipedia.org/wiki/File:People%27s_Action_Party_of_Singapore_logo.svg
```

Comment:
This reference is incorrect (it belongs to another module, although the concerns about its free usage are true). Please remove it.